lib/hex_web/web/router.ex

```diff
@@ -162,6 +163,15 @@ defmodule HexWeb.Web.Router do
     end)
   end
 
+  defp snippet_version({:ok, version}) do
```

Comment:
What do you suggest as an alternative, seeing as you wanted to have the function accept the output of `Version.parse`?